### DISCUSSION TITLE
Split slider tick in results screen back into slider tick and slider end

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#25989] [Ideas] [answered]
Prior to 2023.1114.0, osu!(lazer) used to count slider ticks and slider ends separately in the results screen, and this was incredibly useful because it effectively functioned as a slider break count. However, after that osu!(lazer) update, both slider ticks and slider ends got combined into a single slider tick statistic, and I think that this was a bad change. Let me show a score I set before that update where having them being split would be useful:

Score before 2023.1114.0 (slider ticks and slider ends are separated):

![image](https://github.com/ppy/osu/assets/74088315/9585eabd-d8c6-4f9a-9b58-e012be0b7678)

Score after 2023.1114.0, what it looks like if I watch the replay on the current version of osu!(lazer) (slider ticks and slider ends are combined):

![image](https://github.com/ppy/osu/assets/74088315/96661a51-0da1-44c3-ad02-f73b5b5b541c)

In both cases, you can tell that the score is a play where I sliderbroke, as it has 0 misses but I am still missing a lot of max combo. However, only on the old update can you tell that because I missed 2 slider ticks, I actually sliderbroke twice. On the current version of osu!(lazer) with both slider ticks and slider ends being combined, you cannot tell that this is the case. 

In many cases on osu!stable, it is impossible to tell if someone sliderbroke or just missed a ton of slider ends. osu!(lazer) had already solved this problem by showing both slider ticks and slider ends on the results screen. But after 2023.1114.0, they both got combined and as of now, osu!(lazer) suffers from the same problem that osu!stable does. The combined slider tick statistic on the results screen is pretty much useless as it's impossible to know how many of those are sliderbreaks from missing actual slider ticks and how many are slider end misses.

**Comment by peppy** [comment, marked as answer]
it's no longer possible since we fucked the scoring to match people's stable expectations. lazer is no longer the improvement we hoped for
